lib/store/application.ts

```diff
@@ -32,6 +45,35 @@ export const createApplication = createAsyncThunk(
   }
 );
 
+export const createVerifyCode = createAsyncThunk(
+  'auth/create',
+  async (application: Application) => {
+    const request: CreateAuthRequest = {
+      email: application.mainApplicant?.contactInformation?.emailAddress ?? '',
```

Comment:
So the `?`s here are a bit of a code smell. I made that MainApplicant type to help with this. The MainApplicant type has assurances over the email address being present I think. Although actually there's a good chance you want a string argument here for the email address rather than needing the whole application passed in.

Reply:
Also I'd recommend trying to be consistent in those naming conventions and maybe stick to `emailAddress` everywhere?